sched/scheduler/stateful_scheduler.go

```diff
@@ -414,9 +427,12 @@ addLoop:
 		select {
 		case newJobMsg := <-s.addJobCh:
 			receivedJob = true
-			js := newJobState(newJobMsg.job, newJobMsg.saga)
+			js := newJobState(newJobMsg.job, newJobMsg.saga, s.taskDurations)
 			s.inProgressJobs = append(s.inProgressJobs, js)
 
+			// TODO(jschiller): associate related tasks, i.e. task retries that decorate the taskId?
```

Comment:
There is a lot going on in sq latency... I too was thinking we should measure how long tasks are waiting to start.